scan_promela/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+name = "main"
+version = "1.87.0"
+edition = "2024"
+
+[build-dependencies]
+cfgrammar = "0.13"
+lrlex = "0.13.10"
+lrpar = "0.13.10"
+
+[dependencies]
+cfgrammar = "0.13"
+lrlex = "0.13.10"
+lrpar = "0.13.10"
+regex = "1"
+scan_core = {git = "https://github.com/convince-project/scan.git", package="scan_core"}
+rand = { version = "0.9.1", features = ["small_rng"] }
+log = "0.4"
+
+[lib]
```

Comment:
Moving `lib.rs` and `main.rs` to `src`, the `[lib]` and `[[bin]]` sections should not be needed anymore.